### PR TITLE
Fix screenshot passing key

### PR DESCRIPTION
**Pull request by aaronfay**
Fixes regression from #10 where we moved from using a bash script to our offical Node.js client, but missed converting the ability to pass based on screenshot comparison.

### CODE REVIEW

File: includes/bin/runghostinspectorsuite [hunk 1]
```diff
@@ -96,8 +99,8 @@ echo "Using JSON params: $GI_PARAMS_JSON"
 # Execute the suite
 set +e
 OUTPUT=$(GI_API_KEY=$GI_API_KEY & /bin/run-suite.js $GI_SUITE "$GI_PARAMS_JSON")
-
 PASSING=$?
+# echo $OUTPUT
```
Comment (justinklemm) [inline includes/bin/runghostinspectorsuite:103]: Is this commented out because it's for debugging?

Comment (aaronfay) [reply includes/bin/runghostinspectorsuite:103]: Yeah, I can take it out if you want

**Comment by aaronfay** [comment]
@justinklemm can you hold up on review for a bit, I think I'm missing a piece from this from last week...

**Comment by aaronfay** [comment]
Okay no we're good 😅  I thought I missed the actual comparison, but it's there, review away 👍🏻

File: includes/bin/runghostinspectorsuite (hunk 1 above)
Comment (justinklemm) [reply includes/bin/runghostinspectorsuite:103]: Nah, it's fine. Just wasn't sure if you meant to have it there.

**Review by justinklemm — APPROVED** [review]

**Pull request merged**